packages/client/wallets/aa/src/CrossmintAASDK.ts

```diff
@@ -60,32 +63,31 @@ export class CrossmintAASDK {
 
             const userIdentifier = parseUserIdentifier(user);
 
+            const publicClient = createPublicClient({
+                chain: getViemNetwork(chain as EVMBlockchainIncludingTestnet),
+                transport: http(getBundlerRPC(chain)),
+            });
+
             const owner = await createOwnerSigner({
-                userIdentifier,
-                projectId: this.projectId,
                 chain,
                 walletConfig,
-                crossmintService: this.crossmintService,
             });
 
-            const address = await owner.getAddress();
-
-            const zDevProvider = await ZeroDevEthersProvider.init("ECDSA", {
-                projectId: getZeroDevProjectIdByBlockchain(chain),
-                owner,
-                opts: {
-                    providerConfig: {
-                        rpcUrl: getUrlProviderByBlockchain(chain),
-                    },
-                    paymasterConfig: {
-                        policy: "VERIFYING_PAYMASTER",
-                    },
+            const address = owner.address;
+
+            const ecdsaValidator = await signerToEcdsaValidator(publicClient, {
+                signer: owner,
+            });
+
+            const account = await createKernelAccount(publicClient, {
+                plugins: {
+                    sudo: ecdsaValidator,
                 },
             });
 
-            const evmAAWallet = new EVMAAWallet(zDevProvider, this.crossmintService, chain);
+            const evmAAWallet = new EVMAAWallet(account, this.crossmintService, chain, publicClient, ecdsaValidator);
 
-            const abstractAddress = await evmAAWallet.getAddress();
+            const abstractAddress = evmAAWallet.kernelClient.account!.address;
```

Comment:
can we remove the `!`?
```suggestion
            const abstractAddress = evmAAWallet.kernelClient.account.address;
```